lib/test.js

```diff
@@ -152,15 +152,15 @@ class Test {
 	}
 
 	countPassedAssertion() {
-		if (this.finishing) {
+		if (this.finished) {
 			this.saveFirstError(new Error('Assertion passed, but test has already ended'));
 		}
 
 		this.assertCount++;
 	}
 
 	addPendingAssertion(promise) {
-		if (this.finishing) {
+		if (this.finished) {
 			this.saveFirstError(new Error('Assertion passed, but test has already ended'));
```

Comment:
I realize this is not from this PR, but a pending assertion technically hasn't passed yet. This message should be different.